catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/geofeature/PolygonFeature.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.geofeature;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PolygonFeature extends Feature {
+    private List<Coordinate> coordinates;
+
+    public PolygonFeature() {
+        this.type = "polygon";
```

Comment:
That sounds better for this implementation, yeah - `type` is only a property at all because the JSON serializer being used doesn't seem to support non-properties. 
I'm thinking to avoid it entirely I'll just give the base Feature a getDataObject() method and let subclasses specify their data to be serialized as a return value.